data/hai/unfettered side missions.txt

```diff
@@ -65,3 +66,403 @@ mission "Unfettered returning home"
+			`	However, the paper also mentions you need to have at least ten million credits in cash! You'll have to save up or take a loan and then come back another day before heading to the specified location.`
+				to display
+					"credits" < 10000000
+				defer
+			`	Will you go there?`
+			choice
+				`	(Sure.)`
+				`	(Maybe another time.)`
+					defer
+			label go
+			`	While you're pondering whether or not there will be enough space to land your ship at the beach, you notice a train ticket attached to the back of the paper. A few more written notes give specific directions on how to get there, with a clear demand not to take your starship at the bottom.`
+			`	To your surprise, the local train station is located underground. You can't begin to make sense of the enormity of it - it almost feels like you can reach the entire planet from here. You ask a Hai for directions, feeling embarrassed when you learn that all you had to do was scan your ticket at a nearby terminal and all relevant information would be displayed for you.`
+			`	After departing the train station, you hike for a few hours until you reach your destination. Upon arriving, you look around for the stranger. Besides a cave, some sparse vegetation, and of course the beach itself, you cannot see anything significant in the vicinity. You double-check that these are the right coordinates and look around some more.`
+			`	You wait for some time until you spot two hooded figures further down the beach, and you start to doubt whether or not this was a good idea as they make their way toward you. Even when they're close enough to speak, their faces somehow remain impossible to glimpse beneath the hoods. "Hello, <last>," one says to you. "We are sorry for the secrecy, but the truth is, we cannot afford to take risks. It is best for all if you do not know who we are, and we needed a secluded area to speak to you."`
+			choice
+				`	"Tell me more."`
+					goto "tell me"
+				`	"How do you know me?"`
+					goto know
+				`	(Run away.)`
+			`	They seem surprised at your sudden change of mind, but make no effort to stop you.`
+			label "train back"
+			`	Without any incidents, you return to the train station and buy a ticket back to the spaceport.`
+				decline
+			label know
+			`	The second one grins. "Any outsider landing on our homeworld must be well-known. We know you have helped our kind."`
+			`	"Though do not think we are unaware of your skirmishes with the Unfettered," the first adds with a serious tone.`
+				to display
+					"reputation: Hai (Unfettered)" < -1250
+			label "tell me"
+			`	Suddenly, you hear a loud roar coming from the sea behind you, and the ground begins to shake.`
+			choice
+				`	(Turn around.)`
+			`	The waves become agitated and wind rushes past you as a shape surrounded by fumes emerges out of the water, as if an ancient sea monster was rising from the depths.`
```

Comment:
```suggestion
			`	The waves become agitated and wind rushes past you as a shape surrounded by fumes emerges out of the water, as if an ancient sea monster is rising from the depths.`
```

Reply:
euh idk I feel like it's "was" and I asked tibetiroka they seem to agree, maybe we can have other opinions on it?

Reply:
In terms of grammar, is would be the correct word because the situation would necessitate present tense. (It being a thing that is happening right now.)
However, was feels better to me as well. In my mind, the past tense lends itself to the description of the ship as ancient, even if it doesn't make sense grammatically.
(Though in that case maybe "were," as in "as though an ancient sea monster were rising from the depths," would be better?)